### PR TITLE
Fix #14: Remove helpers and update docs

### LINKED ISSUE #14: .NET 5 does not support @helper directives
How to reproduce:
1. Copy the example from here
https://github.com/Geta/geta-optimizely-extensions#menulist-extension-for-htmlhelper
2. Paste it in any view of .NET 5 project
3. Try to build the project and see the error:

image

### PR DESCRIPTION
**Pull request by brianweet**
Updated the readme to include working examples
Updated sandbox app to use our own extension method for rendering the menu

### CODE REVIEW

**Comment by brianweet** [comment]
Created a separate issue for other broken functionality/examples: https://github.com/Geta/geta-optimizely-extensions/issues/16

File: README.md [hunk 1]
```diff
@@ -41,11 +41,10 @@ _MenuList_ extension method has three optional parameters:
 
 Below is an example how menu can be created for pages under start page.
 
-    @helper MenuItemTemplate(MenuItem item)
-    {
-        <li class="@(item.Selected ? "active" : null)">
+    @{
+        Func<MenuItem, HelperResult> menuItemTemplate =@<li class="@(item.Selected ? "active" : null)">
```
Comment (marisks) [inline README.md:45]: Could it be possible to write it as a local method? Just curious

```suggestion
        HelperResult GetMenuItemTemplate(MenuItem menuItem) {
            return @<li class="@(item.Selected ? "active" : null)">
                @Html.PageLink(item.Page)
            </li>;
```

Comment (brianweet) [reply README.md:45]: Doesn't look like it:
image

Only something like this but don't think it's particularly useful here:
```csharp
void RenderMenuItemTemplate(MenuItem menuItem)
{
    <li class="@(menuItem.Selected ? "active" : null)">
            @Html.PageLink(menuItem.Page)
    </li>
}
```

Comment (brianweet) [reply README.md:45]: Seems they do some rewriting magic:
image

**Review by marisks — APPROVED** [review]

**Pull request merged**